src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -624,6 +624,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\InterfaceTypeAttribute.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\LCIDConversionAttribute.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\LayoutKind.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\Marshal.cs" />
```

Comment:
Could you please add temporary `Condition="'$(TargetsCoreRT)' != 'true'"` for now next to all `Marshal.*` files added here. Getting this to land in ProjectN will require non-trivial changes that I do not want to just amend to the mirror PR.